File: twiml/voice/transcription/transcription-intelligenceservice/transcription-intelligenceservice.php
<?php
require_once './vendor/autoload.php';
use Twilio\TwiML\VoiceResponse;

$response = new VoiceResponse();
$start = $response->start();
$start->transcription(['statusCallbackUrl' => 'https://example.com/your-callback-url', 'intelligenceService' => 'GAaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa']);

echo $response;

File: twiml/voice/transcription/transcription-intelligenceservice/transcription-intelligenceservice.py
from twilio.twiml.voice_response import VoiceResponse, Start, Transcription

response = VoiceResponse()
start = Start()
start.transcription(
    statusCallbackUrl='https://example.com/your-callback-url',
    intelligenceService='GAaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')
response.append(start)

print(response)
